pkg/http/handler/model.go

```diff
@@ -0,0 +1,17 @@
+package httphandler
+
+import (
+	"github.com/google/uuid"
+)
+
+type Item struct {
+	id uuid.UUID
+}
+
+type Error struct {
+	Message string `json:"message,omitempty"`
+}
+
+func (e Error) Error() string {
+	return e.Message
+}
```

Comment:
Model and method may be separated.
Also Item can be general model if you use it as return from GRPC service.

Reply:
So far I have separate pull requests. I will make the single model when pull-requests are merged.